src/compiler/transformers/declarations.ts

```diff
@@ -1798,7 +1798,14 @@ export function transformDeclarations(context: TransformationContext) {
                         if (shouldStripInternal(m)) return;
                         // Rewrite enum values to their constants, if available
                         const constValue = resolver.getConstantValue(m);
-                        return preserveJsDoc(factory.updateEnumMember(m, m.name, constValue !== undefined ? typeof constValue === "string" ? factory.createStringLiteral(constValue) : factory.createNumericLiteral(constValue) : undefined), m);
+                        const newInitializer = constValue === undefined

```

Comment:
Hate to nit one more time, but can this be reformatted into our typical switch/case form?

Reply:
hm, I don't know what you request here. Each branch~ uses a different kind of check so I can't easily use switch/case here (unless we are talking about our favorite `switch (true)` :P)

Reply:
```ts
const newInitializer = constValue === undefined ? undefined
    : typeof constValue === "string" ? factory.createStringLiteral(constValue)
    : constValue < 0 ? factory.createPrefixUnaryExpression(SyntaxKind.MinusToken, factory.createNumericLiteral(-constValue))
    : factory.createNumericLiteral(constValue);
```

Reply:
Sorry, meant match/case.

Reply:
(Or operators on the right, we're inconsistent)

Reply:
thanks for the clarification, I just pushed out the requested change

Reply:
Thanks, sorry for the churn!